Fluid/Filters/ColorFilters.cs

```diff
@@ -25,6 +26,32 @@ public static FluidValue ToRgb(FluidValue input, FilterArguments arguments, Temp
             return new StringValue($"rgb({rgbColor.R}, {rgbColor.G}, {rgbColor.B})");
         }
 
+        public static FluidValue ToHex(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            var color = input.ToStringValue();
+            if (!color.StartsWith("rgb(") && color.EndsWith(")") && color.Length >= 10)
```

Comment:
I think the test is wrong

Reply:
Ya it's 13 ;) I will fix it in `color_to_hsl` PR

Reply:
even the && seems fishy

Reply:
or you are missing grouping parens

Reply:
No worry, I need to refactor this method in the filter I'm working on, I will keep you updated

Thanks